content/en/blog/_posts/2023-01-15-Vulnerable-Microservices/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: blog
+title: "All Microservices Are Vulnerable"
+date: 2023-01-15
+slug: security-behavior
+---
+
+**Author:**
+David Hadas (IBM Research Labs)
+
+As cyber war games continue to intensify in sophistication, organizations deploying cloud services, continue to grow their cyber investments aiming to produce safe and non-vulnerable services. However, the year-by-year growth in cyber investments, does not result in parallel reduction in cyber incidents. Instead, the number of cyber incidents continue to grow annually. Evidently organizations are fighting a battle they are doomed to fail at. No matter how much effort is made to detect and remove cyber weaknesses from deployed services, it seems offenders have the upper hand.
+
+Considering the current spread of offensive tools, sophistication of offensive players, and ever-growing cyber financial gains to offenders, any cyber strategy that relies on constructing a non-vulnerable, weakness-free service in 2022 is clearly too naïve. It seems the only viable strategy is to:
+
+|         |Admit that your services are vulnerable |
+|  ---    | ---  |
```

Comment:
This is a (misused) table. We should to leave the hold in place until either:

- you (as author) agree to use normal markdown for the article
- we (blog team) agree to make an exception

If you need to chat more about this, you could join the next SIG Docs meeting on Zoom. However, I really hope the idea is obvious: **use plain Markdown** and don't muck around trying to style the article by misusing it.